woocommerce/woocommerce
# Revert "Change Variations form shown in Variations tab when there are no variations created (#36957)"

This reverts commit 2e33922e35cc692a0c6905f3f1f13a841dc86063.

### Submission Review Guidelines:

- I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
- I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
- I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/). 
- Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:



Revert "Change Variations form shown in Variations tab when there are no variations created (#36957)", since it caused [unexpected issues](https://github.com/woocommerce/woocommerce/issues/37761) on 7.6.





### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Create a new product
2. Set it to "Variable"
3. Go to "Variations" tab.
4. Verify that the UI was reverted
Before this PR:
<img width="1151" alt="image" src="https://user-images.githubusercontent.com/13437655/234017108-8b544e31-0b37-4a80-880e-6873323e6310.png">
After this PR:
<img width="1161" alt="image" src="https://user-images.githubusercontent.com/13437655/234017996-81682bd3-2d04-4401-8561-e0034b884bd7.png">

5. Go to the Attributes tab and add two global attributes with "Used for variations" checkbox unchecked.
6. Publish the product
7. Check if no attributes were deleted
